api/v1alpha1/podchaos_types.go

```diff
@@ -19,6 +19,7 @@ import (
 
 // +kubebuilder:object:root=true
 // +chaos-mesh:base
+// +chaos-mesh:oneshot=.Spec.Action==PodKillAction
```

Comment:
`ContainerKill`  also belongs to `oneshot`?

Reply:
Yes! Fixed.